doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+refl_10cm,Radar reflectivity
+max_hail_diam_sfc,Maximum hail diameter at lowest model level
+dkt,Atmospheric heat diffusivity
+dku,Atmospheric momentum diffusivity
+cldfra,Instantaneous 3D cloud fraction
+cnvw,Subgrid scale convective cloud water
+skebu_wts,Perturbation velocity - u component
+skebv_wts,Perturbation velocity - v component
+zmtnblck,Level of dividing streamline
+refdmax,Max hourly 1-kilometer above ground level reflectivity
+refdmax263k,Max hourly -10C reflectivity
+sppt_wts,Perturbation velocity
+shum_wts,Perturbation velocity
+spp_wts_pbl,Stochastic perturbed parameterization planetary boundary layer perturbation wind turbines
+spp_wts_pbl,Stochastic perturbed parameterization surface perturbation wind turbines
+spp_wts_mp,Stochastic perturbed parameterization Moller-Plesset perturbation wind turbines
+spp_wts_gwd,Stochastic perturbed parameterization gravitational waves perturbation wind turbines
+spp_wts_rad,Stochastic perturbed parameterization radial perturbation wind turbines
+spp_wts_cu_deep,Stochastic perturbed parameterization cu deep perturbation wts
+sfc_wts,Perturbation amplitude
+ca1,Cellular automata
+ca_deep,Cellular automaton deep convection
+ca_turb,Cellular automaton turbulence
+ca_shal,Cellular automaton shallow convection
+ca_rad,Cellular automaton radiation
+ca_micro,Cellular automaton microphysics
+lakefrac,Lake fraction
+lakedepth,Lake depth
+T_snow,Temperature of snow on a lake
+T_ice,Temperature of ice on a lake
+use_lake_model,Lake model flag
+lake_is_salty,Lake point is considered salty by CLM lake model
+lake_cannot_freeze,CLM lake model considers the point to be so salty it cannot freeze
+lake_t2m,Temperature at 2 meters from lake model
+lake_q2m,2m specific humidity from lake model
+lake_albedo,Mid-day surface albedo over lake
+lake_h2osno2d,Water equivalent of accumulated snow depth over lake
+lake_sndpth2d,Actual accumulated snow depth over lake in CLM lake model
+lake_snl2d,Snow layers in CLM lake model (treated as integer)
+lake_tsfc,Skin temperature from CLM lake model
+lake_savedtke12d,Top level eddy conductivity from previous timestep in CLM lake model
+lake_ht,Lake height
+zmtb,Height of dividing streamline
+zogw,Height of orographic gravity waves
+zlwb,Height of lamb wave beam
+tau_ogw,Orographic gravity waves vertical medium frequency at launch level
+tau_mtb,Orographic mountain beam integrated flux from surface
+tau_tofd,Turbulent orographic form drag integrated flux from surface
+tau_ngw,Non-stationary gravity waves momentum flux at launch level
+du3dt_pbl_ugwp,U-tendency due to planetary boundary layer physics
+dv3dt_pbl_ugwp,V-tendency due to planetary boundary layer physics
+dt3dt_pbl_ugwp,T-tendency due to planetary boundary layer physics
+uav_ugwp,U-daily mean for unified gravity wave physics
+tav_ugwp,T-daily mean for unified gravity wave physics
+du3dt_ogw,Averaged E-W orographic gravity wave tendency
+du3dt_ngw,Averaged E-W all gravity waves tendency

```

Comment:
Is this specifically for non-stationary gravity wave drag? The `ngw` ending suggests yes, but the comment says GWALL, which suggests all gravity waves.